library/core/src/iter/traits/iterator.rs

```diff
@@ -3057,6 +3058,41 @@ pub trait Iterator {
         Cycle::new(self)
     }
 
+    /// Returns an iterator over `N` elements of the iterator at a time.
+    ///
+    /// The chunks do not overlap. If `N` does not divide the length of the
+    /// iterator, then the last up to `N-1` elements will be omitted.
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// #![feature(iter_array_chunks)]
+    ///
+    /// let mut iter = "lorem".chars().array_chunks();
```

Comment:
```suggestion
    /// let mut iter = "lorem".chars().array_chunks::<2>();
```
I think you missed it?

Reply:
It's not required because of type inference